components/tabs/tab-panel-mixin.js

```diff
@@ -26,7 +26,7 @@ export const TabPanelMixin = superclass => class extends superclass {
 			 */
 			selected: { type: Boolean, reflect: true },
```

Comment:
It looks like both `TabMixin` and `d2l-tab-panel` provide a `selected` property. Which one do we want consumers to use? Existing consumers are obviously setting it on the panel.

Reply:
That's a good question. It would definitely be one or the other. I had been thinking on the `d2l-tab`/`TabMixin` usage as to me that makes sense since that's what you select to display the panel content.

However, since the existing behaviour is to add it to the `tab-panel` there's likely an argument to be made about leaving it there for more ease of migration to the new implementation.